src/controllers.ts

```diff
@@ -24,6 +26,7 @@ import {
 const plane = new Plane();
 const ray = new Ray();
 const vec = new Vec3();
+const vec2 = new Vec3();
```

Comment:
Gotta say, this line confused me to begin with...

Reply:
I had actually thought about making them vecA and vecB because I thought it might look confusing, but then decided I was too lazy :)

Reply:
I renamed it vecb